test/test_ops.py

```diff
@@ -272,6 +273,7 @@ def test_numpy_ref(self, device, dtype, op):
             in ("signal_windows_exponential", "signal_windows_bartlett")
             and dtype == torch.float64
             and "cuda" in device
+            or "cpu" in device
```

Comment:
Should we do this instead?
```suggestion
            and device in ["cuda", "cpu"]
```

Reply:
Thanks for the suggestion, applied the changes.

Reply:
This change cannot skip this test on cuda side. https://hud.pytorch.org/pr/pytorch/pytorch/128682#27568460296. We would like to keep previous code.